docs/ECO-METADATA.md

```diff
@@ -44,3 +44,48 @@ The schema also provides an API endpoint to avoid parsing json, however this end
 curl <http://169.254.169.254/eve/v1/external_ipv4>
 
 192.168.1.10
+
+## Location API endpoint
+
+Meta-data service allows applications to obtain geographic coordinates of the device,
+determined using Global Navigation Satellite Systems. This is available only if device
+has a supported LTE modem with an integrated GNSS receiver (e.g. Sierra Wireless EM7565).
+Standalone GPS receivers are currently not supported.
+
+However, by default EVE does not use the location service of the LTE modem and the location
+information is therefore not available.
+To enable location reporting for the device, the `wwan*` port (corresponding to the LTE modem)
+must be **configured for management** with **location tracking** enabled.
```

Comment:
Why configured for management?

FWIW I was wondering if I can make this work with a cellular modem without a (valid) SIM; should be able to get GPS signals in that case but it doesn't make sense configuring it for management in such a case.

Reply:
Actually I meant that it should not be in PCIBack - i.e. it could be for management, shared between apps or both (but not disabled or directly assigned).
And it seems that if the LTE modem is not activated then it refuses to give location:
```
qmicli -p -d /dev/cdc-wdm0 --loc-start --client-no-release-cid
error: couldn't open the QmiDevice: Operation timed out: device is closed
```
Maybe that is a limitation of the QMI protocol.

Reply:
I have updated the documentation to clarify this.

Reply:
OK